components/TextFields/src/MDCTextInputController.h

```diff
@@ -253,6 +253,34 @@
  */
 @property(class, nonatomic, nullable, strong) UIColor *trailingUnderlineLabelTextColorDefault;
 
+/**
+ Height of the underline when text field is first responder.
+
+ Default is underlineHeightActiveDefault.
+ */
+@property(nonatomic, assign) CGFloat underlineHeightActive;
+
+/**
+ Default value for underlineHeightActive.
+
+ Default is 2.
```

Comment:
Super-nit: Follow Apple's approach of documenting default floats as floating-point literals?

    Default is 2.0.

Reply:
I'll go back thru and do that for all in a nother PR.

Reply:
Sounds good to me.